IFC4x3/Properties/s/SingleChannelPower_25tcvQ$vv7RRphaUrbQGbb/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SingleChannelPower_25tcvQvv7RRphaUrbQGbb" Name="SingleChannelPower" UniqueId="85de6e5a-ff9e-476d-bceb-91ed65690965" PropertyType="p_boundedvalue" PrimaryDataType="IfcPowerMeasure" />
```

Comment:
Can the average not be derived from this?

Reply:
It cannot be simply derived from the maximum and minimum